pkg/master/master.go

```diff
@@ -784,12 +785,14 @@ func (m *Master) expapi(c *Config) *apiserver.APIGroupVersion {
 	controllerStorage := expcontrolleretcd.NewStorage(c.ExpDatabaseStorage)
```

Comment:
Not that you edited this line, but should we change it to scaler instead of controller, since we are creating a scale object?